gatsby/create-changelog-posts.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+
+const get = require('lodash.get');
+
+const getChangelogPostPath = require('../src/utils/get-changelog-post-path');
+
+const { DRAFT_FILTER, CHANGELOG_POST_REQUIRED_FIELDS } = require('./constants');
+const createRedirects = require('./create-redirects');
+
+module.exports = async ({ graphql, actions }) => {
+  const result = await graphql(
+    `
+      query ($draftFilter: [Boolean]!) {
+        allMdx(
+          filter: {
+            fileAbsolutePath: { regex: "/content/changelog/" }
+            fields: { isDraft: { in: $draftFilter } }
+          }
+        ) {
+          nodes {
+            id
+            slug
+            fields {
+              redirectFrom
+            }
+            frontmatter {
+              title
```

Comment:
@vannyle it seems we don't really need `redirectFrom` and `title` fields at changelog entries

Reply:
Removed `redirectFrom` and `title`